packages/dds/merge-tree/src/snapshotLoader.ts

```diff
@@ -29,7 +30,8 @@ export class SnapshotLoader {
         private readonly runtime: IFluidDataStoreRuntime,
         private readonly client: Client,
         private readonly mergeTree: MergeTree,
-        logger: ITelemetryLogger) {
+        logger: ITelemetryLogger,
+        private readonly serializer?: IFluidSerializer) {
```

Comment:
Why is it optional? It feels like it has to be present, otherwise things will fall apart, no?

Reply:
Changed it to required.